developer/build.sh

```diff
@@ -45,4 +46,11 @@ builder_describe_outputs \
 
 builder_run_action configure cp "$KEYMAN_ROOT/common/resources/fonts/keymanweb-osk.ttf" "$KEYMAN_ROOT/developer/src/tike/xml/layoutbuilder/"
 
-builder_run_child_actions clean configure build test
+builder_run_child_actions clean configure build test api
+
+function build_docs() {
```

Comment:
Naming question: does this imply a builder action `build:docs` vs the api action?

Reply:
Yeah, I should call it build_api...